docs/make.jl

```diff
@@ -6,17 +6,18 @@ makedocs(;
     authors="Ole Kröger <o.kroeger@opensourc.es> and contributors",
     repo="https://github.com/Wikunia/Javis.jl/blob/{commit}{path}#L{line}",
     sitename="Javis.jl",
-    format=Documenter.HTML(;
-        prettyurls=get(ENV, "CI", "false") == "true",
-        canonical="https://Wikunia.github.io/Javis.jl",
-        assets=String[],
-    ),
+    #format=Documenter.HTML(;
+    #    prettyurls=get(ENV, "CI", "false") == "true",
+    #    canonical="https://Wikunia.github.io/Javis.jl",
+    #    assets=String[],
+    #),
```

Comment:
Can you clarify why you "deleted" this?

Reply:
I have modified make.jl to run my local build without warnings. I will update the code.

Reply:
Thanks you should still be able to build it locally. Please comment if that doesn't work. It might throw a warning that it doesn't know where to deploy but the build folder should be correct.